app/Config/Routes.php

```diff
@@ -12,12 +14,13 @@
  */
 $routes->get('account/username/exists', [Account::class, 'usernameExists']);
 $routes->get('account/email/exists', [Account::class, 'emailExists']);
-$routes->get('account/roles', [Account::class, 'roles']);
+$routes->get('account/roles', [Account::class, 'roles'], ['filter' => AuthFilter::class]);
 $routes->post('account/register', [Account::class, 'register']);
 $routes->post('account/login', [Account::class, 'login']);
 $routes->post('account/logout', [Account::class, 'logout']);
-$routes->get('account', [Account::class, 'get'], ['filter' => AuthFilter::class]);
 $routes->get('images/(:segment)', [Image::class, 'get']);
 $routes->get('events/(:num)', [Event::class, 'get']);
 $routes->get('events', [Event::class, 'get']);
 $routes->get('health', [HealthCheck::class, 'check']);
+$routes->get('dashboard/admin', [AdminDashboard::class, 'index'], ['filter' => AdminAuthFilter::class]);
```

Comment:
is there a way to get only the current Year and Footer Text?
I only need the full Admin Data in the Dashboard.
And it needs to be possible to get the year and text as a normal viewer.
Because I need to display it.
So an extra route would be nice here.